app/javascript/logs/Logs.vue

```diff
@@ -1,17 +1,14 @@
 <template lang="pug">
-div
-  header.flex.justify-between.p-2(v-if="currentUser")
-    div {{ currentUser.email }}
```

Comment:
The logs app showed the user's email, which would have been redundant with the new universal logged-in header, so we are removing it here.